### DISCUSSION TITLE
NOAA V2 Monitor

### DISCUSSION THREAD

**Discussion by ve3elb** [jekhokie/raspberry-noaa-v2#436] [General]
Hello All. Is there anyway to add a small desktop GUI Monitor for Raspberry NOAA so one can see when, how or what's going on with Raspberry NOAA it would be nice to have something.

**Comment by dom-robinson** [comment]
Are you thinking like Grafana ?

[https://grafana.com/grafana/dashboards](https://grafana.com/grafana/dashboards)
